.github/workflows/python-flake8.yml

```diff
@@ -30,5 +30,5 @@ jobs:
     - name: Lint with flake8
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 flake8-builtins
```

Comment:
This line should stay as it is, the flake8-builtins package adds additional checks that we want to have.

Reply:
ok I'll do that